definitions/json/CaseEvent/CaseEvent-WA-nonprod.json

```diff
@@ -0,0 +1,396 @@
+    "EventEnablingCondition": "managingOffice !=\"Unassigned\"",
+    "SecurityClassification": "Public",
+    "ShowEventNotes": "N",
+    "ShowSummary": "N",
+    "Publish": "Y"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "addAmendJudgment",
+    "Name": "Judgment",
+    "Description": "Add or Amend a Judgment",
+    "DisplayOrder": 26,
+    "PreConditionState(s)": "Accepted;Rejected",
+    "PostConditionState": "Accepted(preAcceptCase.caseAccepted=\"Yes\"):1;Rejected",
+    "CallBackURLAboutToStartEvent": "${ET_COS_URL}/dynamicJudgments",
+    "CallBackURLAboutToSubmitEvent": "${ET_COS_URL}/judgementSubmitted",
+    "EventEnablingCondition": "managingOffice !=\"Unassigned\"",
+    "SecurityClassification": "Public",
+    "ShowEventNotes": "N",
+    "ShowSummary": "N",
+    "Publish": "Y"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "uploadDocumentForServing",
+    "Name": "ET1 serving",
+    "Description": "Upload a Document",
+    "DisplayOrder": 15,
+    "PreConditionState(s)": "Accepted",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public",
+    "EventEnablingCondition": "managingOffice !=\"Unassigned\"",
+    "ShowEventNotes": "N",
+    "CallBackURLSubmittedEvent": "${ET_COS_URL}/et1Serving/submitted",
+    "ShowSummary": "N",
+    "Publish": "Y"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "generateCorrespondence",
+    "Name": "Letters",
+    "Description": "Generate Letters",
+    "DisplayOrder": 30,
+    "PreConditionState(s)": "Accepted;Rejected;Submitted;Vetted;Closed",
+    "PostConditionState": "*",
+    "CallBackURLAboutToStartEvent": "${ET_COS_URL}/dynamicLetters",
+    "CallBackURLAboutToSubmitEvent": "${ET_COS_URL}/generateDocument",
+    "CallBackURLSubmittedEvent": "${ET_COS_URL}/generateDocumentConfirmation",
+    "SecurityClassification": "Public",
+    "ShowEventNotes": "N",
+    "ShowSummary": "N",
+    "Publish": "Y"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "caseTransferDifferentCountry",
+    "Name": "Case Transfer (Scotland)",
+    "Description": "Transfer case to Scotland",
+    "DisplayOrder": 35,
+    "PreConditionState(s)": "Submitted;Vetted;Accepted",
+    "PostConditionState": "Transferred",
+    "EventEnablingCondition": "caseType !=\"Multiple\"",
+    "CallBackURLAboutToStartEvent": "${ET_COS_URL}/caseTransfer/initTransferToScotland",
+    "CallBackURLAboutToSubmitEvent": "${ET_COS_URL}/caseTransfer/transferDifferentCountry",
+    "SecurityClassification": "Public",
+    "ShowEventNotes": "N",
+    "EndButtonLabel": "Transfer Case",
+    "ShowSummary": "Y",
+    "Publish": "Y"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "disposeCase",
+    "Name": "Close Case",
+    "Description": "Close Case",
+    "DisplayOrder": 40,
+    "PreConditionState(s)": "Accepted;Rejected",
+    "PostConditionState": "Closed",
+    "CallBackURLAboutToStartEvent": "${ET_COS_URL}/aboutToStartDisposal",
```

Comment:
Nothing to do with WA, but disposeCase doesn't have a Callback on Scotland. Is that expected.

Reply:
Confirmed by Pupinder to be correct